### ISSUE TITLE
Investigate empty Families added by IUCN

### ISSUE THREAD

**Issue by camiplata** [CatalogueOfLife/xcol#123]
For example:
- Ifritidae
- Modulatricidae
- Pityriasidae

<img width="803" alt="Captura de pantalla 2024-06-10 a la(s) 1 29 31 p m" src="https://github.com/CatalogueOfLife/xcol/assets/5700527/14aad58e-bb83-4154-822d-5ac9d65c7f84">

The value of IUCN is mainly synonyms and endemic species, we may consider mergin only from genus and below and not families

**Comment by DianRHR** [comment]
The last extended release merged 280 families from IUCN. I checked 10 different families randomly and any of them had children below. In some cases the base COL includes a subfamily with the "corresponding name", for example Annielinae instead of Annielidae, Dynastinae instead of Dynastide.  Other cases were derived from a typo in the IUCN name.
 Also, some  genus were merged but in a duplicate way as they include a different author. 
Therefore, I consider that we should only merge names of species and below, because the higher taxonomy in this dataset is quite outdated of with errors.
